lib/msal-browser/src/app/Configuration.ts

```diff
@@ -54,20 +56,24 @@ export type Configuration = {
     system?: BrowserSystemOptions
 };
 
+// Default auth options for browser
 const DEFAULT_AUTH_OPTIONS: BrowserAuthOptions = {
     clientId: "",
+    clientSecret: "",
```

Comment:
I would even consider renaming this to something like `_temp_clientSecret` to make it very clear this is temporary.

Reply:
done